Makefile

```
@@ -86,6 +86,14 @@ build-edgecluster-compact:
 	cd ${PWD}/hack/deploy-hub-local && \
 	./build-edgecluster.sh  $(PULL_SECRET) $(OCP_VERSION) $(ACM_VERSION) $(ODF_VERSION) compact
 
+build-edgecluster-sno-dual:
```

```suggestion
build-edgecluster-sno-2nics:
```

What do you think about this name to make it more explicit?

---

makes sense